src/common/ceph_context.cc

```diff
@@ -606,7 +606,8 @@ CephContext::CephContext(uint32_t module_type_,
     crush_location(this),
     _cct_perf(NULL)
 {
-  _log = new ceph::logging::Log(&_conf->subsys);
+  _log = new ceph::logging::Log(&_conf->subsys,
```

Comment:
whether we write the given log messages to `stderr` is configured by `err_to_stderr`, if we don't like this behavior, we can always disable it with appropriate settings. but i don't think we should patch the code for this purpose.